app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] @AlaaAlsalem as previously suggested kindly also add the cases associated with the foreign key.

Reply:
Well resolved 👍🏽